reset.py

```diff
@@ -162,6 +162,13 @@ def transition_reset(self):
             date = min([m.effective_date or today for m in self.confirm.moves]
                 + [production.effective_date or today])
 
+            has_fifo = hasattr(Move, 'fifo_quantity')
+            if has_fifo:
+                cursor.execute(*move_h.update(
+                    columns=[move_h.fifo_quantity],
+                    values=[Move.default_fifo_quantity()],
+                    where=sql_where))
+
             for product in products:
                 Product.__queue__.recompute_cost_price([product], start=date)
```

Comment:
In this scenario, call the function call recompute_cost_price() to update the cost_price. This action will recalculate the cost_price for those specific transactions where the current fifo_quantity becomes 0.